packages/admin/src/utils/callServiceMethod.ts

```diff
@@ -71,6 +73,15 @@ export default async function callServiceMethod<
       };
     }
 
+    // If we want to display the Multiple Editors error page
+    if (err?.response?.data?.error?.message === 'MULTIPLE_EDITORS') {
+      const applicationId = err.config.url
+        .split('/application-forms/')
+        .pop()
+        .split('/')[0];
```

Comment:
not entirely happy with either `err?.response?.data?.error?.message` or `err.config.url.split().pop().split()[0]` so happy to hear suggestions